Exercise_B/index.css

```diff
@@ -2,3 +2,33 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+#container {
+    margin: 10px;
+    height: 200px;
+    width: 600px;
+}
+
+#blue {
```

Comment:
You could technically save some lines here by adding in a class that contains the height and width
Apply that class to each div.